### ISSUE TITLE
Cracked?

### ISSUE THREAD

**Issue by BradTheGoose** [Marekkon5/mmlpi#9]
Can I use cracked accounts? or do I need be premium and login to play?

**Comment by NikoHD203** [comment]
Yes, just tried it, this is cracked, just type in your username. BUT this is illegal, Mojang (obviously) doesn't allow that, it is not allowed to share Mojangs files, like the game, resources etc.
I also tried to create an installer for the version 1.5.2, was finished completely but was locked on the rpi forum because it was a cracked version
From the [Minecraft EULA](https://account.mojang.com/documents/minecraft_eula):

> ONE MAJOR RULE
>
>The one major rule is that you must not distribute anything we've made unless we specifically agree to it. By "distribute anything we've made" what we mean is:
>
>    give copies of our Game to anyone else;
>    make commercial use of anything we've made;
>    try to make money from anything we've made; or
>    let other people get access to anything we've made in a way that is unfair or unreasonable;
>
>unless we specifically agree to it. And so that we are crystal clear, "the Game" or "what we have made" includes, but is not limited to, the client or the server software for our Game and includes Minecraft and Minecraft: Java Edition on all platforms. It also includes updates, patches, downloadable content, add-ons, or modified versions of a Game, part of those things, or anything else we've made.
>
>Otherwise we are quite relaxed about what you do - in fact we really encourage you to do cool stuff - but just don't do those things that we say you can't.
